### ISSUE TITLE
Issues

### ISSUE THREAD

**Issue by acabal** [drgrigg/suetonius_the-lives-of-the-caesars_j-c-rolfe#1]
- `se lint` has a complaint and `se modernize-spelling` makes a lot of changes. For the translator block, see the Enchiridion colophon for an example.

- I see some instances of blockquotes that have the z3998:verse semantic, but the child p tags don't have spans. For example book-2.xhtml line 531.

- I'm not sure if the top-level structure of the chapters is correct. This case is similar to Gallic War, where each "book" is semantically a chapter. So, what we should do is, the top level section (book-1, book-2, etc.) should have the semantic of "chapter" and there should be no child section. In this case we can keep the word "book" in the title, and the h3 becomes the subtitle instead. See Gallic War for example. This will also make the ToC make a lot more sense.

- The `roman` class is not necessary. We can target it with `section > ol` instead and remove the class completely.

- Additionally, we don't want margins between the paragraphs. Is there a way to get text-indent on the li markers? Maybe you can override the core rule that gives text-indent:0 for p:first-child?

- Should we use Arabic numbers for the paragraph numbers? Roman numbers over 100 can be a chore to parse for a reader. And weren't the scans Arabic?

- In endnote #6, the dash in `(89‑88 BC)` is actually a "no-break hyphen" which I assume was put there by the original transcriber. This is preventing `se typogrify` from changing it to an en dash. Can you replace all instances of "no-break hyphen" with a regular hyphen, then run `se typogrify` to fix up what's left over?

- In the endnotes, when a chapter is referenced we should not italicize it (as it's a chapter and not a literal "book" in the way we mean); and the href should cover the book name as well, not just the numbered section. So we want something like: `<abbr>Cf.</abbr> note on <a href="book-1.xhtml#book-1-24">Julius, <span epub:type="z3998:roman">XXIV</span>.1</a>.`

- In endnote backlinks, we now use `epub:type="backlink"` instead of `epub:type="se:referrer"`

- long description: Please use American-style double quotes and put punctuation inside quotation marks

**Comment by drgrigg** [comment]
> se lint has a complaint 
Sorry about that. I discovered that my se toolset was out of date (had a bit of a struggle to update it, to be honest. And this lint error wasn't showing up until I updated.
> se modernize-spelling makes a lot of changes
Oh gosh, I think I spent so much time on cleaning up the source HTML that I clean forgot to run this. Done no.

**Comment by drgrigg** [comment]
> I'm not sure if the top-level structure of the chapters is correct.
The trouble with this is that Books 7 & 8 both include three divisions, three lives, each: Book 7 has Galba, Otho and Vitellius; and Book 8 has Vespasian, Titus and Domitian. So I've made each Life a chapter section. I agree it looks odd in the earlier books where there is one life per book. Can you look at the structure of the later books and let me know what you advise?

**Comment by drgrigg** [comment]
> Should we use Arabic numbers for the paragraph numbers? Roman numbers over 100 can be a chore to parse for a reader. And weren't the scans Arabic?

No, the Loeb edition has Roman numerals for the chapters/subchapters/divisions, whatever we're calling them.
However, I'm perfectly open to changing the list numbering back to Arabic; but then I think I should change every reference in the endnotes to match, ie, change 'See Julius XXV.1' to 'See Julius 25.1'. Do you agree?

**Comment by drgrigg** [comment]
> Long description: Please use American-style double quotes and put punctuation inside quotation marks

Done.

**Comment by drgrigg** [comment]
> The roman class is not necessary. We can target it with section > ol instead and remove the class completely.

The trouble is, this selector also targets endnotes.xhtml. However, if we're dropping the Roman numerals on the subchapters and you're happy with the default list indenting, etc, then we can drop the class. But if we want any special treatment of the lists in the body of the work we'll have to come up with a cleverer selector.

**Comment by drgrigg** [comment]
> Can you replace all instances of "no-break hyphen" with a regular hyphen, then run se typogrify to fix up what's left over?

Done.

**Comment by acabal** [comment]
Re. chapter/book, we can update the items with one chapter to be structured as I suggested. The rest we can leave as you did them. I think that makes semantically even though they're all called "Book".

Re. selectors, you can use: `section[epub|type~="chapter"] > ol, section[epub|type~="part"] > ol{}`

Re. Roman vs Arabic, I think it would be much easier for a modern reader to just have Arabic considering the numbers go so high. I made the same choice for _Beyond Good and Evil_ which is numbered up to >250. Using Arabic for numbers that high is just easier to read. You may be able to script this easily by combining sed/awk and the `se roman2dec` command. Or maybe a small Python script. If it's too much work then don't worry about it and I'll put it on my to do list.

**Comment by drgrigg** [comment]
> In the endnotes, when a chapter is referenced we should not italicize it (as it's a chapter and not a literal "book" in the way we mean); and the href should cover the book name as well

Done.

**Comment by drgrigg** [comment]
> I see some instances of blockquotes that have the z3998:verse semantic, but the child p tags don't have spans.

Done.

**Comment by drgrigg** [comment]
> Re. Roman vs Arabic

Changing the Roman numerals for the list items in the books is easy, just a matter of removing the additional CSS I added.

As for the endnotes, I have some code which will do this (in Swift, haven't yet converted it to Python).

**Comment by drgrigg** [comment]
`section[epub|type~="chapter"] > ol > li > p{`
`	text-indent: 1em;`
`}`

...seems to do the trick for the para indenting in the book list items.

**Comment by drgrigg** [comment]
> I'm not sure if the top-level structure of the chapters is correct. This case is similar to Gallic War, where each "book" is semantically a chapter. 

I think this is now all fixed. I've also tweaked the CSS regarding the text indenting of the 'list' items.

**Comment by drgrigg** [comment]
> use epub:type="backlink" instead of epub:type="se:referrer"

Done.

**Comment by drgrigg** [comment]
I think this is now all done. However, lint is now throwing errors for the first six books, wanting the title tag to be "Chapter 1: The Deified Julius", etc, rather than "Book 1: The Deified Julius". But I really don't want to do this, and of course if I change Books 7 & 8 lint will complain about that, because they contain chapters within them. Do I just ignore lint in this instance?

**Comment by drgrigg** [comment]
No, sorry. I still need to change all the Roman numeral references in the endnotes. Will do that tonight.

**Comment by drgrigg** [comment]
OK, that's all done now. The only remaining question is what I should do about the errors being thrown by lint.

**Comment by acabal** [comment]
You can ignore the errors, we're doing the right thing. I'll review this a little later tonight. Thanks!

**Comment by drgrigg** [comment]
Did a more careful search this morning and fixed some remaining roman numeral references I had missed in the endnotes.
